src/System.Private.CoreLib/shared/System/Double.cs

```diff
@@ -44,6 +44,22 @@ namespace System
         // We use this explicit definition to avoid the confusion between 0.0 and -0.0.
         internal const double NegativeZero = -0.0;
 
+        [NonVersionable]
```

Comment:
This does not need `[NonVersionable]` attribute. It is internal method.

Reply:
Do we even need these two methods? They are called from one place only, and that method has a very detailed knowledge of the bits anyway.

Reply:
Strictly speaking, no. So I'll just move them next to their use location for now.

However, I may open an API proposal suggesting they be exposed. As they are more generally useful when trying to do performance oriented or low-level operations with these types.